### ISSUE TITLE
tracker.events value in custom action is not having the conversation history when using the button actions

### ISSUE THREAD

**Issue by anoop2503** [RasaHQ/rasa#3457]
**Rasa version**:
rasa-core==0.14.3
rasa-core-sdk==0.14.0
rasa-nlu==0.15.0

**Python version**:  Python 3.6.7

**Operating system** (windows, osx, ...):
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS
Release:        18.04
Codename:       bionic

**Issue**:

When I use button actions as interactive components, the tracker.events value in custom actions showing only the conversation history associated with the button actions. All previous conversation history (without having the button actions) is not available in tracker.events.

But, if you send the same intents values as text (instead of clicking buttons), then the complete conversation history is available in tracker.events. That means, the issue is appearing only when using button action. For testing it, you can just print the tracker.events value in the custom action and compare it in both cases (User input as text and user input as button action)

I tested it for a slack bot integration. Please let me know, if you need more details.

**Content of configuration file (config.yml)**:
```yml
language: "en"

pipeline:
- name: "nlp_spacy"                   # loads the spacy language model
- name: "tokenizer_spacy"             # splits the sentence into tokens
- name: "ner_crf"                   # uses the pretrained spacy NER model
- name: "intent_featurizer_spacy"     # transform the sentence into a vector representation
- name: "intent_classifier_sklearn"   # uses the vector representation to classify using SVM
- name: "ner_synonyms"                # trains the synonyms
```

**Content of domain file (domain.yml)** (if used & relevant):
```yaml
utter_check_feedback:
  - text: Was my response helpful?
    buttons:
      - title: "Yes"
        payload: '/response_success'
      - title: "No"
        payload: '/response_failure'
```

**Comment by akelad** [comment]
Thanks for raising this issue, @erohmensing  will get back to you about it soon.

**Comment by erohmensing** [comment]
Hey Anoop! To clarify is this custom action that checks the tracker completely separate from the action where you're uttering the buttons? Could you show me an example story? 

Something else that would be helpful would be if you would run the bot in the command line and let me know if the same problem persists -- that'll help me figure out if the problem is coming from buttons themselves or slack.

**Comment by anoop2503** [comment]
For testing, I just created another bot with minimum knowledge and added the button actions. In this case, the tracker.events in the custom action is showing all the conversation history without any issues. Hence, I believe there could be something wrong in the stories of the other bot, where I found the issue and it may not be an issue with the rasa stack. Will check it myself. Hence, closing the issue.

@akelad, @erohmensing thank you for your prompt response.
